### ISSUE TITLE
[BUG] [WCAG] requests/new - Validation error contrast ratio is too low. 

### ISSUE THREAD

**Issue by j-isenberg** [reach4help/reach4help#645]
**Describe the bug**
The validation error text's colour does not contrast the background enough to meet WCAG compliance. The current contrast is 3.27:1. This is only an acceptable contrast ratio for fonts over 18px. 

Found on https://reach4help-staging-web-client.firebaseapp.com/requests/new 
Possibly an issue on all pages in which validation errors appear. 

**To Reproduce**
1. Navigate to the requests/new page. 
2. Trigger a validation error. 
3. Open console (F12) and turn on the inspector. 
4a (with Axe add-on). Hover over the validation error. Notice that the hover-inspector fails the contrast compliance test. 
4b (without Axe). Click on the validation error. Copy and paste the colour hex codes of the text and the background into a contrast checker (link provided below) [text: FF4D4F, background: ffffff ]
5b. Notice that the contrast ratio fails the contrast compliance test. 

**Expected behavior**
Contrast ratio meets WCAG requirements (4.5:1) of text under 18px on all text. 

**Screenshots**
![2020-06-05_wcag validation error contrast](https://user-images.githubusercontent.com/66037391/83906991-92b8a800-a732-11ea-8ab5-d7df353276ac.png)

**Desktop (please complete the following information):**
 - Windows 10 - Chrome 

**Additional context**
To check contrast: https://contrastchecker.com/
WCAG quick reference on contrast: https://www.w3.org/WAI/WCAG21/quickref/#contrast-minimum

--------
### [ZenHub Information](https://app.zenhub.com/workspaces/reach4help-5e8dcbfb14ac087f410cbabb/issues/reach4help/reach4help/645)

*This information is updated automatically. To modify it, please use ZenHub.*

**Comment by isma-osy** [comment]
Winggo
